controllers/storageconsumer/storageconsumer_controller.go

```diff
@@ -721,6 +729,95 @@ func (r *StorageConsumerReconciler) reconcileNoobaaAccount() error {
 	return nil
 }
 
+func (r *StorageConsumerReconciler) deletionPhase() (reconcile.Result, error) {
+
+	consumerConfigMap := &corev1.ConfigMap{}
+	consumerConfigMap.Namespace = r.namespace
+	consumerConfigMap.Name = cmp.Or(
+		r.storageConsumer.Spec.ResourceNameMappingConfigMap.Name,
+		fmt.Sprintf("storageconsumer-%v", util.FnvHash(r.storageConsumer.Name)),
+	)
+	if err := r.get(consumerConfigMap); client.IgnoreNotFound(err) != nil {
```

Comment:
How can you continue if you didn't find the configmap? Also, what does it mean in relation to deletion if we cannot find a config map?